cypress/support/step_definitions/organisation/create/edit/edit.ts

```diff
@@ -0,0 +1,66 @@
+/// <reference types="cypress" />
```

Comment:
This file should be in `cypress/support/step_definitions/organisation/edit/edit.ts`)

Reply:
any reasons for repeating edit/edit, wouldn't `cypress/support/step_definitions/organisation/edit.js` be better?

Reply:
It is repetitive; unfortunately, it's also necessary for the way the preprocessor is configured. More exactly, it just loads all the scripts in /create/edit/*.{js,ts}. So the file could be called `index.ts`, `junk.ts`, etc. `edit/edit.ts` just makes things clear (so it has the same name as the feature file.